apps/jsii-docgen/src/docgen/schema.ts

```diff
@@ -463,6 +463,11 @@ export interface DocsSchema {
    * Inflight client
    */
   readonly inflight?: string;
+
+  /**
+   * skipping documentation
+   */
+  readonly skipDocs?: boolean;
```

Comment:
@staycoolcall911 and I actually thought about it, the problem with the positive annotation is that we need to find a way to enforce it, otherwise contributors (and code reviewers) might forget, and we might have some things un-documented... 
I do like the `@wingType`/`@wingBuiltIn` annotation :) Can you please elaborate more on the second example? I feel like there is something that I'm missing

Reply:
> the problem with the positive annotation is that we need to find a way to enforce it

I think this problem is mostly avoidable because there aren't going to be that many `@wingType`s. I think it'd at least as likely to forgot to add a negative annotation.

> Can you please elaborate more on the second example?

In wing, the types: `duration`, `num`, `str`, `bool`
do not match the actual class name in `std` when used in wing. To use them for static method access, you use the lowercase/short version e.g. `num.fromJson()` but currently you're generating `Number.fromJson()`